File: app/src/test/java/com/shalzz/attendance/DataAPITest.java
package com.shalzz.attendance;

import com.shalzz.attendance.data.remote.DataAPI;

import org.junit.Test;
import static junit.framework.Assert.assertEquals;

public class DataAPITest {

    @Test
    public void ApiEndpointIsCorrect() {
        assertEquals(DataAPI.Companion.getENDPOINT(), "https://academics.8bitlabs.tech/v2/prod/");
    }
}

File: app/src/test/java/com/shalzz/attendance/DataAPITest.kt
package com.shalzz.attendance

import com.shalzz.attendance.data.remote.DataAPI

import org.junit.Test
import junit.framework.Assert.assertEquals

class DataAPITest {

    @Test
    fun ApiEndpointIsCorrect() {
        assertEquals("https://academics.8bitlabs.tech/v3/prod/", DataAPI.ENDPOINT)
    }
}
